packages/AreaChart/src/AreaChart.tsx

```diff
@@ -387,14 +384,26 @@ const AreaChart: React.FunctionComponent<AreaChartProps> = ({
         stroke: "none",
         fill: "none"
     };
-
     const buildAreaDefs = (): JSX.Element => {
         return (
             <defs>
-                <linearGradient id="curveAreaFillGradient" x1="0" y1="0" x2="0" y2="1">
-                    <stop stopColor={variables.electricBlue50} />
-                    <stop offset="78%" stopColor="#EDF6FF" />
-                    <stop offset="96%" stopColor={variables.samoyed} />
+                <linearGradient 
+                    id="lineGradient" 
+                    x1="0" 
+                    y1="0" 
+                    x2="0"
+                    y2="96%"
+                    gradientUnits="userSpaceOnUse"
+                >
+                    <stop stopColor="#47A584" />
```

Comment:
Where are these colors coming from?

Reply:
All data tokens are from here https://www.figma.com/file/he7mOkVFtqk0m3dgWKLcXk/%F0%9F%90%B8-Data-visualization?type=design&node-id=40%3A1101&mode=design&t=aWERRvcA3IcZ0HRa-1 , which we don't have on our side yet.